### PR TITLE
apps/filters: make sure ordering also works on annotated fields

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

**Comment by fuzzylogic2000** [comment]
Same here! Done! I will try to add tests, though... @Rineee @goapunk

File: adhocracy4/filters/widgets.py [hunk 1]
```diff
@@ -61,6 +61,12 @@ def render(self, name, value, attrs=None, choices=(), renderer=None):
         })
 
 
+class OrderingWidget(DropdownLinkWidget):
+    """Used in .filters.OrderingFilter"""
```
Comment (Rineee) [inline adhocracy4/filters/widgets.py:65]: This should be DynamicChoicesOrderingFilter, right?

File: adhocracy4/filters/filters.py [hunk 2]
```diff
@@ -75,6 +78,27 @@ def filter(self, qs, value):
...
+
+    def filter(self, qs, value):
+        qs = qs.annotate_positive_rating_count() \
+            .annotate_negative_rating_count() \
+            .annotate_comment_count()
+        if value == ['-comment_count']:
+            return qs.order_by('-comment_count')
+        elif value == ['-positive_rating_count']:
+            return qs.order_by('-positive_rating_count')
+
```
Comment (Rineee) [inline adhocracy4/filters/filters.py:98]: Not sure, but maybe do we want a short comment, why we are annotating here? Just so we remember in the future..

**Review by Rineee — APPROVED** [review]
Yes, supercool! :partying_face:  Just two tiny things about comments..

File: adhocracy4/filters/widgets.py (hunk 1 above)
Comment (fuzzylogic2000) [reply adhocracy4/filters/widgets.py:65]: Right, good catch! I renamed the filter while moving and added the new comment at the same time and then did not update that...

File: adhocracy4/filters/filters.py (hunk 2 above)
Comment (fuzzylogic2000) [reply adhocracy4/filters/filters.py:98]: Yes! Good idea!

**Pull request merged**